spec/fixtures/test-config.xml

```diff
@@ -82,6 +82,7 @@
     <preference name="orientation" value="portrait" />
     <icon id="icon" src="icon.png" />
     <icon height="255" id="logo" src="logo.png" width="255" />
+    <platform name="android"></platform>
```

Comment:
Why the extra android here?

Reply:
That is what causes the bug described in CB-13979, although I agree it's a weird scenario.

If I add the empty platform tag without the rest of these changes, existing tests fail.
With these changes, they pass even when the empty platform tag is present.

Reply:
All tests pass with the empty platform tag removed.

With these changes, all tests also pass with the empty platform tag added.

Reply:
Sounds good. Feel free to merge if you feel good about it!